engine/src/main/java/io/zeebe/engine/processing/streamprocessor/writers/Writers.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.0. You may not use this file
+ * except in compliance with the Zeebe Community License 1.0.
+ */
+package io.zeebe.engine.processing.streamprocessor.writers;
+
+/** Convenience class to aggregate all the writers */
+public final class Writers {
```

Comment:
Personally I find the name a little non-descriptive. On the other hand it is nice and short. And I don't have a better name right now. If you can think of a more descriptive name, then go for it. Otherwise just ignore this comment.

Reply:
I prefer shorter names that work within certain contexts (like the specific package in this case, or short field names in the context of the class' name)